platform/core/src/services/MeasurementService/MeasurementService.ts

```diff
@@ -549,26 +554,27 @@ class MeasurementService extends PubSubService {
       );
     }
 
+    const oldMeasurement = this.measurements[internalUID];
+
     const newMeasurement = {
+      ...oldMeasurement,
```

Comment:
This is where the isSelected gets set - along with finding, findingSites, site, ...